hypervisor/arch/x86/sched.S

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+    .text
+
+   .code64
+   .align       8
+   .global      arch_switch_to
+arch_switch_to:
```

Comment:
Add an description for the API.
BTW, how does KVM do? I remember it uses inline ASM too.

Reply:
Will add the description. For KVM, it actually is using Linux kernel process scheduler which using task structure memory to save/restore context instead of stack.